src/phoenix/server/api/openapi/main.py

```diff
@@ -1,6 +1,32 @@
+import json
+from argparse import ArgumentParser
+from enum import Enum
+from typing import Optional, Tuple
+
 from .schema import get_openapi_schema
 
+
+class Format(Enum):
+    READABLE = "readable"
+    COMPRESSED = "compressed"
+
+
 if __name__ == "__main__":
-    import yaml  # type: ignore
+    parser = ArgumentParser()
+    parser.add_argument(
+        "--format",
+        type=Format,
+        choices=[Format.READABLE, Format.COMPRESSED],
+        required=True,
+        help='The format of the OpenAPI schema ("readable" or "compressed").',
+    )
+    args = parser.parse_args()
 
-    print(yaml.dump(get_openapi_schema(), indent=2))
+    indent: Optional[int] = None
+    separator: Optional[Tuple[str, str]] = None
+    format = args.format
+    if format is Format.READABLE:
+        indent = 2
+    else:
+        separator = (",", ":")
```

Comment:
any reason to not just output it readable?

Reply:
I was initially having difficulty getting the OpenAPI schema to generate on Pydantic V1, so I exported it as a JSON file using Pydantic V2 and bundled it with the rest of our static assets (hence, wanted a compressed version). This turned out to be unnecessary because I eventually got the schema generation working on Pydantic V1, but left this code in case it turns out to be useful.